tycho-extras/tycho-pomless/src/main/java/org/eclipse/tycho/pomless/TychoBundleMapping.java

```diff
@@ -115,7 +111,7 @@ protected void initModel(Model model, Reader artifactReader, File artifactFile)
 
     @Override
     protected Properties getEnhancementProperties(File file) throws IOException {
-        //Look up build.properties in the project's root. The passed file points to the 'META-INF' folder.
+        //Look up build.properties in the project's root. The given file points to the 'META-INF' folder.
```

Comment:
maybe one should check here:

1. name of directory is META-INF
2. file is a directory

Reply:
Done.